Setup your first development environment.md

```diff
@@ -0,0 +1,83 @@
+# Setup your first development environment
+
+To be able to learn programming you will need to have some programs that make your life easier.
+No reasonable developer will want to write code inside of Notepad or Word, or share code throught USB drives.
+In this chapter we will help you setup your development environment.
+
+
+
+## Your IDE
+
+An IDE comes in all sorts and flavours and as of late there are a few really solid options.
+The top 3 IDE's for C# currently are:
+
+- [Visual Studio](https://www.visualstudio.com/)
+- [Visual Studio Code](https://www.visualstudio.com/)
```

Comment:
Link should be
https://code.visualstudio.com/

Also does code come with a CLR, or does that need to be installed separately?

Reply:
From what i can see Code does come with a CLR but if not this is specified in the installation instructions that i link to a bit further down.